docs/HyperSync/hypersync-url-endpoints.md

```diff
@@ -7,61 +7,65 @@ slug: /hypersync-url-endpoints
+| Fantom | 250 | https://fantom.hypersync.xyz or https://250.hypersync.xyz | bronze |   |
+| Flare | 14 | https://flare.hypersync.xyz or https://14.hypersync.xyz | bronze |   |
+| Fuji | 43113 | https://fuji.hypersync.xyz or https://43113.hypersync.xyz | gold |   |
+| Gnosis | 100 | https://gnosis.hypersync.xyz or https://100.hypersync.xyz | gold |   |
+| Gnosis Chiado | 10200 | https://gnosis-chiado.hypersync.xyz or https://10200.hypersync.xyz | bronze |   |
+| Goerli | 5 | https://goerli.hypersync.xyz or https://5.hypersync.xyz | bronze |   |
+| Harmony Shard 0 | 1666600000 | https://harmony-shard-0.hypersync.xyz or https://1666600000.hypersync.xyz | bronze |   |
+| Holesky | 17000 | https://holesky.hypersync.xyz or https://17000.hypersync.xyz | gold |   |
+| Inco Gentry Testnet | 9090 | https://inco-gentry-testnet.hypersync.xyz or https://9090.hypersync.xyz | bronze |   |
+| Kakarot Sepolia | 1802203764 | https://kakarot-sepolia.hypersync.xyz or https://1802203764.hypersync.xyz | bronze |   |
+| Kroma | 255 | https://kroma.hypersync.xyz or https://255.hypersync.xyz | bronze |   |
+| Linea | 59144 | https://linea.hypersync.xyz or https://59144.hypersync.xyz | gold |   |
+| Lukso | 42 | https://lukso.hypersync.xyz or https://42.hypersync.xyz | bronze |   |
+| Manta | 169 | https://manta.hypersync.xyz or https://169.hypersync.xyz | bronze |   |
+| Mantle | 5000 | https://mantle.hypersync.xyz or https://5000.hypersync.xyz | gold |   |
+| Metis | 1088 | https://metis.hypersync.xyz or https://1088.hypersync.xyz | bronze |   |
+| Mev Commit | 17864 | https://mev-commit.hypersync.xyz or https://17864.hypersync.xyz | bronze |   |
+| Moonbeam | 1284 | https://moonbeam.hypersync.xyz or https://1284.hypersync.xyz | gold |   |
+| Neon Evm | 245022934 | https://neon-evm.hypersync.xyz or https://245022934.hypersync.xyz | bronze |   |
+| Optimism | 10 | https://optimism.hypersync.xyz or https://10.hypersync.xyz | gold |   |
+| Optimism Sepolia | 11155420 | https://optimism-sepolia.hypersync.xyz or https://11155420.hypersync.xyz | gold |   |
+| Polygon | 137 | https://polygon.hypersync.xyz or https://137.hypersync.xyz | gold |   |
+| Polygon Amoy | 80002 | https://polygon-amoy.hypersync.xyz or https://80002.hypersync.xyz | bronze |   |
+| Polygon Zkevm | 1101 | https://polygon-zkevm.hypersync.xyz or https://1101.hypersync.xyz | gold |   |
+| Publicgoods | 424 | https://publicgoods.hypersync.xyz or https://424.hypersync.xyz | bronze |   |
+| Rsk | 30 | https://rsk.hypersync.xyz or https://30.hypersync.xyz | bronze |   |
+| Scroll | 534352 | https://scroll.hypersync.xyz or https://534352.hypersync.xyz | gold |   |
+| Sepolia | 11155111 | https://sepolia.hypersync.xyz or https://11155111.hypersync.xyz | gold |   |
+| Shimmer Evm | 148 | https://shimmer-evm.hypersync.xyz or https://148.hypersync.xyz | bronze |   |
+| Staging Eth | 1 | https://staging-eth.hypersync.xyz or https://1.hypersync.xyz | gold |   |
```

Comment:
What is Staging Eth

Reply:
Yeah, I'll filter it out :grimacing: It is Ozgurs test/staging instance :+1: